spec/services/api/v2/sections/chapters_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+RSpec.describe Api::V2::Sections::ChaptersController do
```

Comment:
I think you mean for this file to be under spec/requests/api/v2/sections/chapters_controller_spec.rb?